.github/pull_request_template/bugfix.md

```diff
@@ -0,0 +1,26 @@
+# 🐛 Bug Fix
+
+## Checklist
+- [ ] Issue: #___
+- [ ] Branch: `patch-[issue]-[desc]`
+- [ ] Title: `<component>: <message> Fix #<issue>`
+- [ ] Added regression test
+- [ ] Doesn't break existing functionality
```

Comment:
I think this is potentially a bit hard to quantify for the contributor. Ideally, I guess that if the CI passes, the contributor should assume that no existing functionality is broken, but I'm not sure if it makes sense having a bullet point for this.